src/test/java/com/homework/lesson11/FruitsShopTest.java

```diff
@@ -0,0 +1,77 @@
+package com.homework.lesson11;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.homework.lesson11pr.Fruits;
+import com.homework.lesson11pr.FruitsShop;
+import com.homework.lesson11pr.FruitsType;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.List;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class FruitsShopTest {
+
+    ObjectMapper objectMapper;
+    private FruitsShop fruitsShop;
+    private List<Fruits> fruitsList;
+
+    @Before
+    public void setUp() {
+        fruitsShop = new FruitsShop();
+    }
+
+    @Test
+    public void addFruits() {
```

Comment:
add assertion methods here
this test would be passed even addFruits() wasnt working